src/plugins/data/common/query/timefilter/get_time.ts

```diff
@@ -24,7 +24,9 @@ const calculateLowerBound = (from: string, forceNow?: Date): undefined | Moment
 const calculateUpperBound = (to: string, forceNow?: Date): undefined | Moment =>
   dateMath.parse(to, { roundUp: true, forceNow });
 
-const isRelativeTime = (value: string): boolean => value.includes('now');
+const isRelativeTime = (value: string | Moment): boolean => {
```

Comment:
Where is the `Moment` value coming from?
I see this function only used in `createTimeRangeFilter` function with `TimeRange` `from`/`to` values, which are both strings. Am I missing something here?

Reply:
There is definitely a mismatch between the compile & run-time types for `TimeRange` inside src/plugins/data/public/query/timefilter/timefilter.ts. I've opened https://github.com/elastic/kibana/issues/160824 to track this cleanup.